bazel/macros/build_yarn_workspace.bzl

```diff
@@ -0,0 +1,37 @@
+"""
+Builds Yarn Workspaces via Babel and makes it ready for other repositories.
+"""
+
+# TODO: https://github.com/bazelbuild/rules_nodejs/commit/1e357fdad4527477e34065ab9c880753522a4c41
+load("@build_bazel_rules_nodejs//internal/js_library:js_library.bzl", "js_library")
+load("@npm//@babel/cli:index.bzl", "babel")
+
+# TODO: verify validity of `@generated` headers while building the project
+
+def build_yarn_workspace(name, package_name, deps = []):
+    babel(
+        name = "%s.build" % name,
+        outs = ["dist"],
+        args = [
+            "$(execpath src)",
+            "--out-dir",
+            "$(@D)/dist",
+            "--ignore",  # TODO: do not ignore tests but run the compiled versions (?)
+            "src/**/*.test.js",
+        ],
+        data = [
+            ":package.json",
+            ":src",
+            "//:babel.config.js",
+            "//:package.json",
+            "@npm//@adeira/babel-preset-adeira",
+        ],
+    )
```

Comment:
We are not building flow and es-modules outputs?

Reply:
Not yet. But I want to figure out how to do it before considering this to be stable. It should not be part of this build step I think. This build is mainly for other parts of the monorepo to be able to use the built versions of our packages.

Reply:
Sure, makes sense, one way is to build `lib` folder and `es` folder, and have main and module point to the respective folders, but I don't know how to put in the flow files yet 🤕

Reply:
Stay tuned, we are going to figure it out. I also don't know how to do it quite yet.